packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <div>
```

Comment:
Should we remove this root div as the root element can now be the `p` tag

Reply:
good spot!